ciao-launcher/main.go

```diff
@@ -369,6 +385,23 @@ DONE:
 			if err != nil {
 				break DONE
 			}
+			clusterConfig, err := client.conn.ClusterConfiguration()
+			if err != nil {
+				glog.Errorf("Unable to get Cluster Configuration %v", err)
+				break DONE
+			}
+			computeNet = clusterConfig.Configure.Launcher.ComputeNetwork
+			mgmtNet = clusterConfig.Configure.Launcher.ManagementNetwork
+			diskLimit = clusterConfig.Configure.Launcher.DiskLimit
+			memLimit = clusterConfig.Configure.Launcher.MemoryLimit
+			err = startNetwork(doneCh)
+			if err != nil {
+				glog.Errorf("Failed to start network: %v\n", err)
```

Comment:
I think it would also be useful to add a log statement that outputs the values of all the configuration variables to the launcher logs.